File: algorithms/c/richestCustomerWealth.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize){
    int i,j;
    int sum = 0;
    int maxWealth = 0;
    for (i=0;i<accountsSize;i++){
        sum = 0;
        for(j=0;j<accountsColSize[i];j++){
            sum = sum + accounts[i][j];
        }
        if (sum > maxWealth){
            maxWealth = sum;
        }
    }
    return maxWealth;
}

File: algorithms/python3/richestCustomerWealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        maxWealth=0
        for x in range(len(accounts)):
            sum = 0
            for y in range(len(accounts[x])):
                sum += accounts[x][y]
            if (maxWealth<sum):
                maxWealth=sum
        return maxWealth
